<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QLabel" name="bookLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>КНИГИ</string>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ПОЛЬЗОВАТЕЛИ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="giftBook">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Отдать книгу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeBooks">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Управление книгами</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeUsers">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>560</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Управление пользователями</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>361</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="booksTable">
      <column>
       <property name="text">
        <string>Название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Автор</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Число страниц</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>361</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="usersTable">
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Отчество</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>№ читательского</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>changeUsers</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onManageUsersClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeBooks</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onManageBooksClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>giftBook</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>giftBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onManageUsersClicked()</slot>
  <slot>onManageBooksClicked()</slot>
  <slot>giftBook()</slot>
 </slots>
</ui>
